controllers/users.js

```diff
@@ -1029,6 +1030,17 @@ const updateUsernames = async (req, res) => {
   }
 };
 
+const getUsersWithAbandonedTasks = async (req, res) => {
+  try {
+    const data = await userService.getUsersWithIncompleteTasks();
+    if (data.length === 0) res.status(204).send();
```

Comment:
why this?

Reply:
This is the case which we were discussing yesterday in hangout, if the response is empty then send 204.

Reply:
The response is not empty. It has a list which is empty right?

Reply:
Yes, the response previous was
```
{
  "message": "No users with abandoned tasks found",
  "data": []
}
```
Since there is no data to send to the user, I have replaced it with 204 based on yesterday's discussions.